apps/condo/domains/user/schema/AuthenticateUserWithPhoneAndPasswordService.js

```diff
@@ -13,8 +14,34 @@ const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
 
 const redisGuard = new RedisGuard()
 
-const GUARD_WINDOW_SIZE_SEC = 60 * 60 // seconds
-const GUARD_WINDOW_LIMIT = 10
+const GUARD_DEFAULT_WINDOW_SIZE_SEC = 60 * 60 // seconds
+const GUARD_DEFAULT_WINDOW_LIMIT = 10
+
+/**
+ * @typedef {Object} TAuthGuardQuota
+ * @property {number} windowSizeSec The window size in seconds
+ * @property {number} windowLimit Attempts limit during the window
+ */
+
+/**
+ * @type {Record<string, TAuthGuardQuota>}
+ *
+ * Possible values:
+ * 1. Change all
+ * { "i.p.v.4": { windowSizeSec: 3600, windowLimit: 60 } }
```

Comment:
Still not clear enough. Consider `x.x.x.x` notation, or `127.0.0.1`, or something like `<ip_address>`

Reply:
`i p v 4` - is not clear? :)
What are the other meanings?

Reply:
idk, `h.t.t.p`, or `i.p.v.6`. 

Your dot-separated protocol is not what I expect form documentation (and I read a lot of them daily)